modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -852,12 +852,8 @@ def _take_2d_positional(
                         row_internal_indices, col_internal_indices
                     )
                     for col_idx, col_internal_indices in col_partitions_dict.items()
-                    if isinstance(col_internal_indices, slice)
-                    or len(col_internal_indices) > 0
```

Comment:
why?

Reply:
This is necessary so as not to filter out dataframes if we call `_take_2d_positional` function with parameters `row_positions==[]` or `col_positions==[]`.

Reply:
I see. Makes more sense now.